scripts/src/main/java/com/mathewsachin/fategrandautomata/scripts/modules/Support.kt

```diff
@@ -393,4 +409,36 @@ class Support(fgAutomataApi: IFGAutomataApi) : IFGAutomataApi by fgAutomataApi {
         val mlbSimilarity = prefs.support.mlbSimilarity
         return limitBreakRegion.exists(limitBreakPattern, Similarity = mlbSimilarity)
     }
+
+    private fun skillLevels(bounds: Region, skillLevels: List<Boolean>): Boolean {
+        val y = bounds.Y + 325
+        val x = bounds.X + 1600
+
+        val skillLoc = listOf(
+            Location(x + 2 + 25, y),
```

Comment:
Why areou you adding 2 and then 25 instead of doing `x = bounds.X + 1627`?

Reply:
Because I was testing while adjusting the positions. 🤣. I'll change that before merging.

Reply:
Done